docs/content/operator/common/monitoring.mdx

```diff
@@ -0,0 +1,102 @@
+last_executed_checkpoint_age_ms{pct="95"} 221
+last_executed_checkpoint_age_ms{pct="99"} 297
+last_executed_checkpoint_age_ms_count 287523
+last_executed_checkpoint_age_ms_sum 4291692609431
+last_executed_checkpoint_timestamp_ms 1741700144842
+uptime{chain_identifier="2304aa97",process="fullnode",version="0.10.3-rc-c2adf3fd68e9"} 20925
+```
+
+```sh
+curl --json '{"jsonrpc":"2.0","id":1,"method":"iota_getLatestCheckpointSequenceNumber","params":[]}' localhost -s | jq .result
+```
+
+```sh
+"43428454"
+```
+
+### Node Metrics
+
+IOTA nodes expose a wide range of metrics to be scraped by [Prometheus](https://prometheus.io/).
+By default, metrics are available at the `http://localhost:9184/metrics` endpoint.
+The best way to visualize these metrics is to use [Grafana](https://grafana.com/).
+Additionally, a common approach is to use [node exporter](https://github.com/prometheus/node_exporter) to scrape performance metrics from the node and push them to Prometheus.
+
+### Monitoring Services
+
+Implementing monitoring services is essential to ensure the reliability, security, and performance of the blockchain network by providing real-time insights, detecting anomalies, enabling proactive issue resolution, and receiving automatic alerts.
+
+**Prometheus and Grafana**  
+Currently, we do not have an official Grafana setup guide.
+However, you can try out a setup prepared by the community: https://github.com/stakeme-team/grafana-iota.
+
+Also, there is a Grafana setup for the [local private network](https://github.com/iotaledger/iota/tree/develop/dev-tools/grafana-local), which might be a good example of how to build your own setup. Officially supported dashboards can be found [here](https://github.com/iotaledger/iota/tree/develop/setups/grafana) in the IOTA repository.
+
+**Dolphin**  
+Dolphin is a CLI tool that provides high-level features for validator, fullnode, and bridge monitoring. Under the hood, it uses the IOTA node Prometheus metric exporter to check the health of the node. More info: https://gitlab.com/blockscope-net/dolphin-v2.
+
+### Logs
+Logs are controlled using the `RUST_LOG` environment variable. 
+The `RUST_LOG_JSON=1` environment variable can optionally be set to enable logging in JSON structured format.
+
+Depending on your deployment method, these are configured in the following places:
+
+- If you are using Systemd natively, [here](https://github.com/iotaledger/iota/blob/develop/setups/fullnode/systemd/iota-node.service).
+- If you are using Docker Compose, [here](https://github.com/iotaledger/iota/blob/develop/setups/validator/docker_old/docker-compose.yaml).
+
+To view and follow the IOTA node logs:
+
+```sh
+$ journalctl -u iota-node -f
+```
+
+To search for a particular match:
+```sh
+$ journalctl -u iota-node -g <SEARCH_TERM>
+```
+
+#### Docker Logs
+If you are using Docker Compose, by default, logs are stored at `/var/lib/docker/containers/[container-id]/[container-id]-json.log`.
```

Comment:
I never cared about the logs folder. The following commands are the only way people use it IMO.

Reply:
Removed in 0c0a6337ba4b90bca65d1713222fc05d4843e3bf